apps/be-api/src/app/modules/products/routes/products.routes.ts

```diff
@@ -0,0 +1,131 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { idParamSchema } from '../../../shared/validations';
+import { ProductCreateDTO, ProductUpdateDTO } from '../dto';
+import { ProductService } from '../services';
+import { productSchema } from '../validations';
+
+interface ProductParams {
+  id: string;
+}
+
+export async function productRoutes(fastify: FastifyInstance) {
```

Comment:
```suggestion
export async function productAdminRoutes(fastify: FastifyInstance) {
```

also rename file name